models/capital_v2.json

```diff
@@ -157,5 +157,28 @@
             "column": "amount",
             "label": "Amount"
         }
+    },
+    "can_aggregate": "True",
```

Comment:
actually it's that it can't aggregate - can_aggregate should be false, and it should be a boolean, not a string.

Also it's not the cube that can/can't aggregate, it's the dimension. So the right place for a property like this is `dimensions.capital_type`